miband.py

```diff
@@ -267,8 +267,10 @@ def send_custom_alert(self, type, phone):
                 base_value = '\x03\x01'
         svc = self.getServiceByUUID(UUIDS.SERVICE_ALERT_NOTIFICATION)
         char = svc.getCharacteristics(UUIDS.CHARACTERISTIC_CUSTOM_ALERT)[0]
-        char.write(bytes(base_value+phone,'utf-8'), withResponse=True)
-
+        # 3 new lines: space for the icon, two spaces for the time HH:MM
+        text = base_value+phone+'\x0a\x0a\x0a'+msg.replace('\\n','\n')
```

Comment:
this is the heart of the change, as the user can't input new lines in a way the band will ingest.